examples/7guis/timer.slint

```diff
@@ -15,6 +15,7 @@ MainWindow := Window {
 
     VerticalBox {
         HorizontalBox {
+            padding-left: 0;
```

Comment:
That's a Box-in-Box anti-pattern.
Maybe it might be better using a Grid. Or use Layout 
But i guess that's good enough like you did.

(We should perhaps consider coalescing the padding in the Box tough.)